### PR TITLE
Fix tracker support overlaps

### LINKED ISSUE #208: Fix outer tracker service routing overlaps
The outer tracker service routing overlaps the pfRICH (and probably mRICH) in the negative end, and the 90 degree turn conflicts with the imaging ecal in the positive end. I suggest (pragmatically) to:
* shorten second projective cone on the negative end to the face of the pfRICH, and have a discontinuous step to a cylinder outside of the pfRICH/mRICH
* replace 90 degree turn with an extension of the cone near the dRICH in the positive end for the dRICH end

### PR DESCRIPTION
**Pull request by sly2j**
Remove outer detail from forward tracker support, and make backward tracker support discontinuous

Also fixes some of the central detector volume allocations.

Closes #208

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### CODE REVIEW

File: compact/mrich.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <define>
     <constant name="MRICH_rmin"   value="8*cm"/>
     <constant name="MRICH_rmax"   value="min(63*cm, BackwardPIDRegion_rmax)"/>
-    <constant name="MRICH_length" value="BackwardPIDRegion_length"/>
+    <constant name="MRICH_length" value="BackwardPIDRegion_length-28.87*cm"/>
```
Comment (wdconinc) [inline compact/mrich.xml:6]: This I don't understand.

Comment (sly2j) [reply compact/mrich.xml:6]: This shifts the mrich back by the difference in length between the mrich and the pfrich to ensure they are both placed at the back of the PID region. Not sure how else to do this, except to maybe introduce an extra parameter in the top level definitions  file (which would be a bit cleaner)

Comment (c-dilks) [reply compact/mrich.xml:6]: I should double-check the `dawn` views again, but I think we resolved the pfRICH/mRICH placement in #127 (just look at `definitions.xml` and `{pf,m}rich.xml`, there are several workaround/test commits in that PR that need to be reverted)

Comment (wdconinc) [reply compact/mrich.xml:6]: > This shifts the mrich back by the difference in length between the mrich and the pfrich

That's where I am confused: the difference is 11.4 cm (pfRICH 54.1 cm - mRICH 42.7 cm), not 28.87 cm.

Comment (c-dilks) [reply compact/mrich.xml:6]: Let's resolve this in #158

Comment (wdconinc) [reply compact/mrich.xml:6]: Really? I'm just done here.

**Pull request merged**